essential-tests/src/testFixtures/kotlin/org/stellar/anchor/platform/integrationtest/PlatformApiTests.kt

```diff
@@ -27,7 +27,8 @@ import org.stellar.anchor.platform.AbstractIntegrationTests
 import org.stellar.anchor.platform.TestConfig
 import org.stellar.anchor.util.GsonUtils
 
-@Disabled
+@Disabled // TODO: https://stellarorg.atlassian.net/browse/ANCHOR-693
```

Comment:
Can we put a little details here so that we don't need to go back to the ticket?

Reply:
Agree, let's do this little TODOs later on to not forget things that were put away!